manim/scene/graph_scene.py

```diff
@@ -1120,3 +1120,54 @@ def update_func(group, alpha):
         )
         secant_slope_group.kwargs["x"] = target_x
         secant_slope_group.kwargs["dx"] = target_dx
+
+    def interpolate_between_points(self, pt1, pt2):
+        """Interpolates the space between two points by returning a function for a straight line between them.
+
+        Parameters
+        ----------
+        pt1 : Tuple[Union[:class:`int`, :class:`float`], Union[:class:`int`, :class:`float`]]
+            The first point to calculate the line througn.
+
+        pt2 : Tuple[Union[:class:`int`, :class:`float`], Union[:class:`int`, :class:`float`]]
+            The second point to calculate the line througn.
+
+        Returns
+        -------
+        :class:`function`
+            The function of a straight line passing through the two points."""
+        return (
+            lambda x: ((pt2[1] - pt1[1]) / (pt2[0] - pt1[0])) * x
+            + pt1[1]
+            - ((pt2[1] - pt1[1]) / (pt2[0] - pt1[0])) * pt1[0]
+        )
+
+    def get_graphs(self, *args, **kwargs):
+        """Returns a list of the graphs of all functions passed in *args.
+
+        Parameters
+        ----------
+        *args : List[Union[:class:`function`, Tuple[:class:`function`, :class:`str`, ...]]]
+            If a function, the function to graph. If a tuple, the first item must be the function to graph and the rest must be positional arguments for a :meth:`.GraphScene.get_graph` call.
+
+        **kwargs : :class:`dict`
+            Any valid kwargs of a :meth:`.GraphScene.get_graph` call.
+
+        Returns
+        -------
+        :class:`list`
+            List of :class:`.VGroup` instances containing the :class:`.ParametricFunction` instances that make up the connective lines between points."""
```

Comment:
Is this true? It seems this method is more general than that, no?

Reply:
That may have been an accidental copy-paste from the original `get_graphs_from_points` method. I'll fix it.